core/driver/test-plugin/src/main/scala/org/virtuslab/ideprobe/test/OpenWindowAction.scala

```diff
@@ -8,6 +8,7 @@ import javax.swing.Icon
 
 class OpenWindowAction extends AnAction("Open Window") with DumbAware {
   override def actionPerformed(e: AnActionEvent): Unit = {
+    val p = e.getProject
```

Comment:
Huh I'd be really surprised if that was the reason 😯

Reply:
Yeah, but test failures are so confusing that I want to try something. Looks like something from my changes is the reason, as these tests were successful on previous PR today https://github.com/VirtusLab/ide-probe/pull/240
(still, I have tha same failures for tests locally on **master** branch, but maybe this is some other reason for this)